### DISCUSSION TITLE
Hide display setup on singleton collection

### DISCUSSION THREAD

**Discussion by Nitwel** [directus/directus#3029] [Archive]
What do you think @rijkvanzanten ?

**Comment by benhaynes** [comment]
Interesting. I understand that the singleton collection wouldn't have its fields displayed on a browse page, but technically it could be linked to relationally... and then you would need the displays. Right @rijkvanzanten ?

**Comment by rijkvanzanten** [comment]
What @benhaynes said is correct. Displays are used in other places than just the layout (for example in the title and relational interfaces), so we shouldn't hide the setup
